README.md

```diff
@@ -3194,6 +3195,63 @@ You can configure eventual consistency with the `ClientConfig.properties` below:
 
 - `hazelcast.invalidation.reconciliation.interval.seconds`: Default value is `60` seconds. This is a periodic task that scans cluster members periodically to compare generated invalidation events with the received ones from the client Near Cache.
 
+### 7.8.3. Pipelining
+
+With the pipelining, you can send multiple parallel requests with a defined concurrency limit without waiting for their results.
+
+The pipelining allows to control back pressure, i.e., to limit 
+the number of inflight requests, and it also provides a convenient way to wait for all the results.
+
+```javascript
+function createLoadGenerator(map) {
+    var counter = 0;
+    return function() {
+        if (counter < 100) {
+            return map.get(counter++);
+        }
+        return null;
+    }
+}
+
+Client.newHazelcastClient().then(function (client) {
+    var map;
+    return client.getMap('pipelining').then(function (mp) {
+        map = mp;
+        var pipelining = new Pipelining(10, createLoadGenerator(map), true);
+        return pipelining.run();
+    }).then(function (results) {
+        // Wait for completion
+        console.log(results);
+        return client.shutdown();
+    });
+});
+```
+
+In the above example, we make 100 concurrent `map.get()` calls, but the maximum number of inflight calls is 10.
+
+By increasing the depth of the pipelining, throughput can be increased. 
+
+The pipelining has its own back pressure, you do not need to enable the back pressure on the member to have this feature on the pipelining. 
+However, if you have many pipelines, you may still need to enable the member back pressure because it is possible to overwhelm the system with requests in that situation.
```

Comment:
.... to enable the back pressure on the member since the system may become overwhelmed by the requests in that situation.